src/scipp/core/bins.py

```diff
@@ -355,7 +356,7 @@ def concat(self, dim: Optional[str] = None) -> Union[_cpp.Variable, _cpp.DataArr
             All bins along `dim` concatenated into a single bin.
         """
         if dim is not None:
-            return _call_cpp_func(_cpp.buckets.concatenate, self._obj, dim)
```

Comment:
Can `_cpp.buckets.concatenate` be removed now?

Reply:
For now I just remove the Python bindings. My next step will be to look into `groupby.bins.concat` which uses related code on the C++ side and hopefully more cleanup can be done after that.